teeracle/src/mock.rs

```diff
@@ -120,6 +120,7 @@ impl timestamp::Config for Test {
 parameter_types! {
 	pub const MomentsPerDay: u64 = 86_400_000; // [ms/d]
 	pub const MaxSilenceTime: u64 = 172_800_000; // 48h
+	pub const MaxOracles: u32 = 2;
```

Comment:
rather `MaxWhitelistLength`?
and set it higher as 2 is very limiting for tests. maybe 10?

Reply:
This is a param that has to be configured in the Node runtime. Here again, I think that MaxWhitelistLength is not so "user-friendly". If "oracle" is ok, then what about MaxTrustedOracles ?

Reply:
What about this?

Reply:
MaxWhitelistedReleases